index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CapStone</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div class="nav-header">
+            <ul>
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a>Logout</a>
+                <a>Mypage</a>
+                <a>Bollywood</a>
+            </ul>
+        </div>
+        <nav class="nav">
+            <div class="nav-header2">
+                <i id="menu" class="menu fa-sharp fa-solid fa-bars"></i>
+                <label class="logo">
+                    <h3>SoundHound</h3>
+                    <h2>Music Fest</h2>
+                </label>
+                <ul id="menu-icon">
+                    <li class="cross-icon"><i id="cross-icon" class="cross fa-solid fa-x"></i></li>
+                    <li class="nav"><a href="./index.html">Home</a></li>
+                    <li class="nav"><a href="./about.html">About</a></li>
+                    <li class="nav"><a href="#event">Event</a></li>
+                    <li class="nav"><a href="#partner">Partner</a></li>
+                    <li class="booktick nav"><a href="#book">Book Ticket(s)</a></li>
+                </ul>
+
+            </div>
+        </nav>
+        <div class="music-section">
+            <p class="heading3">"Hello! HallHulla Music Lover!"</p>
+            <h1 class="heading1">SoundHound HallHulla Music Fistival 2022</h1>
+            <p class="pragraph">The Honky Tonk Festival brings together all country music lovers - the hillbilly music
+                lovers, with the aim to expressing a way to celebrating glorious country heritage, culture and
+                traditions. This fest is meant to rejoice our special moments and emotions in our lives with our loved
+                ones, be it as it may, we're all country boys and girls.</p>
+            <h3 class="date-format">2022.11.11(FRI) ~ 12(SAT)</h3>
+            <p class="pragraph2">@ Ryman Auditorium, Nashville, Tennessee and more</p>
+
+        </div>
+
+    </header>
+
+    <div class="event-cards">
```

Comment:
- [ ] Please consider using a `<section>` element to wrap this section rather than a `<div>`. This is best practice for HTML semantics.